frontend/lib/src/components/elements/DocString/DocString.test.tsx

```diff
@@ -36,14 +37,15 @@ const getProps = (
 
 describe("DocString Element", () => {
   const props = getProps()
-  const wrapper = shallow(<DocString {...props} />)
 
   it("renders without crashing", () => {
-    expect(wrapper).toBeDefined()
+    render(<DocString {...props} />)
+    expect(screen.getAllByTestId("stDocstring")).toHaveLength(2)
```

Comment:
Why is the length 2?

Reply:
Length is 2 because there is another doc at the bottom for `stDocstring-Doc`

Reply:
Fixed! as you said, there was a render in a weird place and that was removed. Thank you!